### DISCUSSION TITLE
When will the coverlet.collector Package With the NewtonSoft Upgrade Change be Deployed

### DISCUSSION THREAD

**Discussion by R-Hurl** [coverlet-coverage/coverlet#1531] [Q&A] [answered]
Just curious when a new package with changes related to the following PR: #1510 will be deployed? Apologies if this has already occurred.

**Comment by Bertk** [comment, marked as answer]
Sorry for the late reply. The PR is meanwhile merged and you can help us for releasing it.

Please use nightly build results and sen us your feedback e.g.

https://dev.azure.com/tonerdo/coverlet/_artifacts/feed/coverlet-nightly/NuGet/coverlet.msbuild/overview/8.0.0-preview.15.g893dcc8fb6

**Comment by Bertk** [comment]
[coverlet.collector](https://www.nuget.org/packages/coverlet.collector) V8.0.0 is released.
